stage_1.md

```diff
@@ -158,6 +158,16 @@ false && echo foo || echo bar
 true || echo foo && echo bar
 ```
 
+This should echo "foobar":
+```
+true && printf foo || false && printf bar
+```
+
+This is equivalent to:
+```
+(((true && printf foo) || false) && printf bar)
```

Comment:
Ok, but we don't want to give misleading syntax around `()` (which are not grouping, but subshell, in sh); maybe we can be explicit about the associativity in another way?

I would suggest `{ { { true && printf foo; } || false; } && printf bar; }` but this is probably even less clear.

Thanks!

Reply:
Yeah, this is a good point. Thinking about it more, maybe the best way is to use parens but also explicitly state that they're meant to indicate associativity, not subshells?

Either that or just remove the attempt to show the associativity, and just use this as an example.